chirp/drivers/retevis_ra25.py

```diff
@@ -0,0 +1,1098 @@
+                                     charset=self.VALID_DTMF,
+                                     mem_pad_char='\n')
+        rs.set_value = set_value2.__get__(rs, RadioSettingValueString)
+        rset = RadioSetting("dtmf_start.id", "PTT ID Starting", rs)
+        dtmf.append(rset)
+
+        _current = ''.join(c for c in str(_dtmf_end.id)
+                           if c in self.VALID_DTMF)
+        rs = RadioSettingValueString(minlength=0, maxlength=16,
+                                     current=_current,
+                                     charset=self.VALID_DTMF,
+                                     mem_pad_char='\n')
+        rs.set_value = set_value2.__get__(rs, RadioSettingValueString)
+        rset = RadioSetting("dtmf_end.id", "PTT ID Ending", rs)
+        dtmf.append(rset)
+
+        _current = ''.join(c for c in str(_remote_kill.id)
+                           if c in self.VALID_DTMF)
+        rs = RadioSettingValueString(minlength=0, maxlength=16,
+                                     current=_current,
+                                     charset=self.VALID_DTMF,
+                                     mem_pad_char='\n')
+        rs.set_value = set_value2.__get__(rs, RadioSettingValueString)
+        rset = RadioSetting("remote_kill.id", "Remote Kill", rs)
+        dtmf.append(rset)
+
+        _current = ''.join(c for c in str(_remote_stun.id)
+                           if c in self.VALID_DTMF)
+        rs = RadioSettingValueString(minlength=0, maxlength=16,
+                                     current=_current,
+                                     charset=self.VALID_DTMF,
+                                     mem_pad_char='\n')
+        rs.set_value = set_value2.__get__(rs, RadioSettingValueString)
+        rset = RadioSetting("remote_stun.id", "Remote Stun", rs)
+        dtmf.append(rset)
+
+        # DTMF Encode List
+        dtmf_mem = RadioSettingGroup("dtmf_mem", "DTMF Encode List")
+        group.append(dtmf_mem)
+
+        for i in range(0, 16):  # M1-16
+            _current = ''.join(c for c in str(
+                _dtmf_list[i].m) if c in self.VALID_DTMF)
+            rs = RadioSettingValueString(minlength=0, maxlength=16,
+                                         current=_current,
+                                         charset=self.VALID_DTMF,
+                                         mem_pad_char='\n')
+            rs.set_value = set_value2.__get__(rs, RadioSettingValueString)
+            rset = RadioSetting(f"dtmf_list[{i}].m", f"M{i+1}", rs)
+            dtmf_mem.append(rset)
+
+        # Immutables Info
+        info_menu = RadioSettingGroup("immut_info", "Radio Info")
+        group.append(info_menu)
+
+        _warning = """
+            To Upload to the Radio this setting must match the Radio Mode.
+            The Radio Mode can be changed by holding down [V/M] during powerup.
+            Note: Changing the Radio Mode will set Radio memory to defaults."""
+
```

Comment:
This will end up with some spaces in places you don't expect when rendered. Can you make this a single string and let the UI format it? So:
```
_warning = ('To upload the ...'
            'The radio ... ',
   ... )
```